claims_hosp/delphi_claims_hosp/backfill.py

```diff
@@ -69,9 +72,57 @@ def store_backfill_file(claims_filepath, _end_date, backfill_dir):
         "/claims_hosp_as_of_%s.parquet"%datetime.strftime(_end_date, "%Y%m%d")
     # Store intermediate file into the backfill folder
     backfilldata.to_parquet(path, index=False)
+    return path
+
+
+def merge_existing_backfill_files(backfill_dir, backfill_file, issue_date, logger):
+    """
```

Comment:
Could you add more explanation to this function? Otherwise people can easily get confused by this one and the function below.

Reply:
Hopefully the explanation I had made sense.